src/Common/ProgressIndicatorAwareTrait.php

```diff
@@ -4,18 +4,31 @@
 trait ProgressIndicatorAwareTrait
 {
     use Timer;
+
+    /**
+     * @var ProgressIndicator
+     */
     protected $progressIndicator;
 
+    /**
+     * @return int
+     */
     public function progressIndicatorSteps()
     {
         return 0;
     }
 
+    /**
+     * @param \Robo\Common\ProgressIndicator $progressIndicator
```

Comment:
There are places that call `setProgressIndicator(null)`; if you are going to typehint this, then we'll need to provide a `clearProgressIndicator()` method.